app/services/feedback/getFeedbackBannerState.ts

```diff
@@ -1,14 +1,20 @@
 import { type Session } from "@remix-run/node";
-import type { BannerState } from "~/components/userFeedback";
+import { BannerState } from "~/components/userFeedback";
 
 export const bannerStateName = "bannerState";
+export const userRatingFieldName = "wasHelpful";
 
 export const getFeedbackBannerState = (session: Session, url: string) => {
-  const getSessionForFeedbackState = session.get(bannerStateName) as Record<
-    string,
-    BannerState
-  >;
-  if (getSessionForFeedbackState) {
-    return getSessionForFeedbackState[url];
-  }
+  const getSessionForFeedbackState =
```

Comment:
It's variable, so better call:

```suggestion
  const feedbackState =
```

or 

```suggestion
  const state =
```

Reply:
i was following the same naming conventions you have used not to look odd, but yeah you are right the naming was not the best, will change it.

Reply:
That naming convention was wrong, my mistake 🥲

Thanks for fixing it!